app/assets/javascripts/actions/notifications.js

```diff
@@ -25,10 +25,23 @@ export const sendSuccessNotification = (message) =>
     }, 4000);
   }
 
-export const sendErrorNotification = (error) =>
-  !!error.response
-    ? sendServerErrorNotification(error)
-    : sendDefaultErrorNotification();
+export const sendErrorNotification = (error, { custom = false } = {}) => {
+  if (!!error.response) return sendServerErrorNotification(error);
+  if (custom) return sendCustomErrorNotification(error);
+  return sendDefaultErrorNotification();
+}
+
+export const sendCustomErrorNotification = code =>
+  (dispatch, getState, { Notification }) => {
```

Comment:
Since you are not using `getState`, you could rename to `_`. So, this way we know you are not using the second param.
```js
(dispatch, _, { Notification }) => {
```
What do you think @talyssonoc ?

Reply:
Ok!